src/component/contents/DraftEditorBlock.react.js

```diff
@@ -31,9 +31,9 @@ const getScrollPosition = require('getScrollPosition');
 const getViewportDimensions = require('getViewportDimensions');
 const nullthrows = require('nullthrows');
 
-import type {BidiDirection} from 'UnicodeBidiDirection';
-import type {DraftDecoratorType} from 'DraftDecoratorType';
-import type {List} from 'immutable';
+import type { BidiDirection } from 'UnicodeBidiDirection';
```

Comment:
Oops, I guess my editor auto-formated some stuff based on Textio's rules. Thanks for that!